quickstart_templates/zero_downtime_deployment/vmss/vmss-blue-green-deploy-job.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<flow-definition plugin="workflow-job@2.16">
+  <displayName>Deploy To VMSS</displayName>
+  <description>Deploy a managed OS image to the VMSS with the blue/green deployment strategy.</description>
+  <keepDependencies>false</keepDependencies>
+  <properties>
+    <hudson.model.ParametersDefinitionProperty>
+      <parameterDefinitions>
+        <hudson.model.StringParameterDefinition>
+          <name>IMAGE_ID</name>
+          <description>The ID of the OS image that needs to be deployed to the production environment.</description>
+          <defaultValue></defaultValue>
+        </hudson.model.StringParameterDefinition>
+      </parameterDefinitions>
+    </hudson.model.ParametersDefinitionProperty>
+  </properties>
+  <definition class="org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition" plugin="workflow-cps@2.42">
+    <script><![CDATA[
+node {
+    def resourceGroup = '{insert-resource-group}'
```

Comment:
Make resourceGroup a parameter?

Reply:
The resource group is tightly bound to the target VMSS. It's not a variable that the user can configure.

I will leave it as it is.